solr/README.md

```diff
@@ -0,0 +1,77 @@
+<!--
+Copyright (c) 2012 YCSB contributors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you
+may not use this file except in compliance with the License. You
+may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing
+permissions and limitations under the License. See accompanying
+LICENSE file.
+-->
+
+## Quick Start
+
+This section describes how to run YCSB on Solr running locally. 
+
+### 1. Set Up YCSB
+
+Clone the YCSB git repository and compile:
+
+    git clone git://github.com/brianfrankcooper/YCSB.git
+    cd YCSB
+    mvn clean package
+
+### 2. Set Up Solr
+
+There must be a running Solr instance with a core/collection already
+defined with an appropriate schema.xml file.
+
+If running in SolrCloud mode ensure there is an external Zookeeper
+cluster running
+
+### 3. Run YCSB
+    
+Now you are ready to run! First, load the data:
+
+    ./bin/ycsb load solr -s -P workloads/workloada -p table=<core/collection name>
+
+Then, run the workload:
+
+    ./bin/ycsb run solr -s -P workloads/workloada -p table=<core/collection name>
+
+For further configuration see below: 
+
+### Defaults Configuration
+The default setting for the Solr node that is created is as follows:
+	
+	'solr.cloud' mode = 'false'
+	'table'/core/collection name = 'ycsb'
+	'solr.host.list' = 'localhost:8983'
+	'zookeeper.host.list' = 'localhost:2181'
+
+### Custom Configuration
+If you wish to customize the settings used to create the Solr node
+you can created a new property file that contains your desired Solr 
+node settings and pass it in via the parameter to 'bin/ycsb' script. Note that 
+the default properties will be kept if you don't explicitly overwrite them.
+
+Assuming that we have a properties file named "myproperties.data" that contains 
+custom Solr node configuration you can execute the following to
+pass it into the Solr client:
+
+    ./bin/ycsb run solr -P workloads/workloada -P myproperties.data -s
+
+If you wish to use SolrCloud mode ensure a Solr cluster is running with an
+external zookeeper cluster and an appropriate collection has been created.
+Make sure to pass the following properties as parameters to 'bin/ycsb' script.
+
+	cloud.mode=true
```

Comment:
This seems to clash with above, is the property ```cloud.mode``` or ```solr.cloud```, The latter is prefered as it refers to the binding it belongs to in the name.